spanner/cloudexecutor/proto/cloud_executor.pb.go

```diff
@@ -0,0 +1,9431 @@
+// Copyright 2023 Google LLC
```

Comment:
This is a 9k lines code file. Can you break this into smaller files where each file is having a single responsibility? It's otherwise difficult to review.